.github/workflows/chromatic.yml

```
@@ -0,0 +1,16 @@
+name: 'Chromatic'
+
+on: push
+jobs:
```

Now we know it works, let's have it run only on main - I think this is the right way to do that
```suggestion
on:
  push:
    branches:
      - main
jobs:
```

---

I wondered about that – we'd lose the nice feature of having a storybook per branch (which seems like it might be nice for QA?)

---

FWIW, Raphaëlle was asking me earlier about per-branch deploys; might be a wider conversation to have.

---

Oh does it do that automatically? Amazing! Yes, let's do it like this then - can you add a comment to this workflow file so we don't forget that reasoning?